src/pybind/mgr/dashboard/module.py

```diff
@@ -397,6 +398,12 @@ def set_mgr_created_self_signed_cert(self):
         if result.retval != 0:
             return result
         return 0, 'Self-signed certificate created', ''
+    
+    @CLICommand("dashboard feedback get")
+    def get_issues_cli(self, issue_number: str):
+        cephTrackerClient = CephTrackerClient()
+        response = cephTrackerClient.get_issues(issue_number)
+        return 0,str(response["issue"]), ''
```

Comment:
Are these the issues reported by the user owning this API key? That might be useful (`https://tracker.ceph.com/issues?author_id=me` is the Web UI URL). Otherwise, I don't think it's useful to provide a generic wrapper for querying Redmine tickets.

Reply:
At first I wrote it just to test if it works, but fetching the user's reported issues sounds like a good idea, I'll make the changes.

Reply:
This URL didn't work for me in UI. But I found the request in [redmine doc](https://www.redmine.org/projects/redmine/wiki/rest_issues) . I'll test it out and post the update here.

Reply:
Following the [docs](https://www.redmine.org/projects/redmine/wiki/rest_issues), get request should be sent [here](https://tracker.ceph.com/issues.json?author_id=me) to get the data whereas the issues are visible (here)[https://tracker.ceph.com/issues?author_id=me]. But similar to the browser, the get request fetches an empty array. Any idea how to implement this?